sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/text/TextFileFormat.scala

```diff
@@ -84,6 +85,21 @@ class TextFileFormat extends FileFormat with DataSourceRegister {
     }
   }
 
+  override def canSplitFiles(files: Seq[Path], conf: Configuration): Boolean = try {
```

Comment:
actually since the implementation is always TextInputFormat, maybe we should just inline TextInputFormat.isSplitable.

Do we need the Hadoop Configuration here? Would be great to do away with it.